src/io/pti.jl

```diff
@@ -943,4 +943,744 @@ function _populate_defaults!(data::Dict)
+    #     end
+    # end
+
+    println(io, "0 / END OF INTER-AREA TRANSFER DATA, BEGIN OWNER DATA")
+
+    # Owner Data
+    if haskey(data, "owner") 
+        for (_, owner) in sort(data["owner"], by = (x) -> parse(Int64, x))
+            # Get Dict in a PSSE way and print it
+            psse_comp = _pm2psse_owner(owner)
+            _print_pti_str(io, psse_comp, _pti_dtypes["OWNER"])
+        end
+    else
+        Memento.warn(_LOGGER, string("Skipping OWNER data because it does not found"))
+    end
+
+    println(io, "0 / END OF OWNER DATA, BEGIN FACTS CONTROL DEVICE DATA")
+
+    Memento.warn(_LOGGER, string("Export FACTS data is not yet supported"))
+
+    println(io, "0 / END OF FACTS CONTROL DEVICE DATA, BEGIN SWITCHED SHUNT DATA")
+
+    # Switched Shunt
+    for (_, shunt) in sort(data["shunt"], by = (x) -> parse(Int64, x))
+        # Skip Fixed Shunts
+        if shunt["source_id"][1] != "switched shunt"
+            continue
+        end
+        
+        # Get Dict in a PSSE way
+        psse_comp = _pm2psse_switched_shunt(shunt)
+
+        # Print it in the file
+        _print_pti_str(io, psse_comp, _pti_dtypes["SWITCHED SHUNT"])
+    end
+
+    println(io, "0 /END OF SWITCHED SHUNT DATA, BEGIN GNE DEVICE DATA")
+
+    Memento.warn(_LOGGER, string("Export GNE data is not yet supported"))
+
+    println(io, "0 /END OF GNE DEVICE DATA")
+    println(io, "Q")
+end
+
+
+"Given a PTI component dict print it in the .raw file"
+function _print_pti_str(io::IO, component, _dtype)
+    str = ""
+    for (data, type) in _dtype
+        if type == String
+            # Avoid losing the trailing whitespace in ID of components
+            str *= "$(component[data]),"
+        else
+            str *= "$(component[data]),\t"
+        end
+    end
+    println(io, str)
+end
+
+
+"""
+Export remaining keys of a componet
+"""
+function _export_remaining!(sub_data::Dict{String, Any}, pm_comp::Dict{String, Any}, pti_default::Dict{String, Any})
+    # Try to get the value from the PM component (is in lower case), else use the default
+    for (key, default) in pti_default
+        if ! haskey(sub_data, key)
+            sub_data[key] = get(pm_comp, lowercase(key), default)
+        end
+    end
+end
+
+
+"""
+Create a header for the case
+"""
+function _pm2psse_header(pm::Dict{String, Any})
+    sub_data = Dict{String, Any}()
+
+    for (dtype, _) in _transaction_dtypes
+        sub_data[dtype] = get(pm, dtype, _default_case_identification[dtype])
+    end
+
+    return sub_data
+end
+
+
+"""
+Parses PM Bus data to PSS(R)E-style.
+"""
+function _pm2psse_bus(pm_bus::Dict{String, Any})
+    sub_data = Dict{String, Any}()         
+    sub_data["I"] = pm_bus["bus_i"]
+
+    if haskey(pm_bus, "name")
+        sub_data["NAME"] = "\'$(pm_bus["name"])\'"
+    elseif haskey(pm_bus, "string")
+        sub_data["NAME"] = "\'$(pm_bus["string"])\'"
+    else
+        sub_data["NAME"] = "\'$(_default_bus["NAME"])\'"
+    end
```

Comment:
why are we supporting the key `"string"` here?  "name" in the PowerModels convention.  So `get`, `name`, `_default_bus["NAME"]` should work.

Reply:
I really don't remember why I did this. But what you say works perfectly.